backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/ApiUserManagementTest.java

```diff
@@ -186,14 +200,51 @@ void addUser_superUser_success(String operation) {
     assertEquals(EnumSet.allOf(UserPermission.class), extractPermissionsFromUser(user));
 
     assertUserCanAccessAllFacilities(user);
+
+    verify(_oktaRepo)
```

Comment:
Yeah, that was my thinking on it. Can't really test the outcome, so this is checking the intent.

More specific arguments would mean we don't have to reset for every test, but in some of these tests those arguments don't seem easy to get.  I kind of like the reset, though, since then we know what is being verified happened in the test and not the initialization code.